### PR TITLE
 docs(atomic): add support for lit-html for additional markup in stories 

### PR DESCRIPTION
**Pull request by olamothe**
This would be the way we can inject "additional markup" for a given story. 

That is: when, for a given Story, we need more than just the component itself, like other components or styling that we need to inject. I've added an example with the Breadbox (adding Facets components).

As explained in scrum last week, we need to either use HTMLElements or an HTML string (that's what works best with Storybook with Stencil today).

However, as pointed out by Sami, using a string for the markup has the drawback of us potentially making mistakes/typo.

I went with the approach of using `lit-html` to write the markup, coupled with `lit-analyzer`, that scan the HTML string passed through the `html` function of lit-html.

It is able to verify everything that is basic HTML as well as our custom components tags (from `dist/types/components.d.ts`). 

However, it is not able to detect Stencil components `@Prop()` annotation/property and map it to "known" HTML attributes.

As such, I had to disable that warning.

https://coveord.atlassian.net/browse/KIT-1160

### CODE REVIEW

**Review by ThibodeauJF — APPROVED** [review]

File: packages/atomic/src/components/atomic-breadbox/atomic-breadbox.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import {html} from 'lit-html';
+
+const {defaultModuleExport, exportedStory} = defaultStory(
+  'Atomic/Breadbox',
+  'atomic-breadbox',
+  {},
+  BreadboxDoc,
+  {
+    additionalMarkup: () =>
+      html`
```
Comment (btaillon) [inline packages/atomic/src/components/atomic-breadbox/atomic-breadbox.stories.tsx:12]: Didn't know this kind of syntax existed in TS/JS. I've only seen that as "macros" in other languages.

**Review by samisayegh — APPROVED** [review]

File: packages/atomic/src/components/atomic-breadbox/atomic-breadbox.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+       <div style="margin:20px 0">Select facet value(s) to see the Breadbox component.</div>
+       <div style="display: flex; justify-content: flex-start; >
```
Comment (btaillon) [inline packages/atomic/src/components/atomic-breadbox/atomic-breadbox.stories.tsx:14]:
```suggestion
      <div style="margin:20px 0">Select facet value(s) to see the Breadbox component.</div>
      <div style="display: flex; justify-content: flex-start;"> 
```

File: packages/atomic/tsconfig.json [hunk 3]
```diff
@@ -11,7 +11,17 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "jsx": "react",
-    "jsxFactory": "h"
+    "jsxFactory": "h",
+    "plugins": [
+      {
+        "name": "ts-lit-plugin",
```
Comment (btaillon) [inline packages/atomic/tsconfig.json:17]: Does this need to be added to dev dependencies? It seems missing from dependencies, right now.

**Review by btaillon — APPROVED** [review]

File: packages/atomic/tsconfig.json (hunk 3 above)
Comment (olamothe) [reply packages/atomic/tsconfig.json:17]: No, not needed.

It's used by the cli tool `lit-analyzer`.

**Pull request merged**